test/basics.jl

```diff
@@ -50,4 +50,10 @@ end
 
     @test GAP.Globals.IdFunc(2^62) isa Int64
 
+    x = GAP.Globals.Indeterminate( GAP.Globals.Rationals )
+    f = x^4+1
+    i1 = GAP.Globals.IdealDecompositionsOfPolynomial( f )
+    i2 = GAP.Globals.IdealDecompositionsOfPolynomial( f , onlyone = true)
```

Comment:
```suggestion
    i2 = GAP.Globals.IdealDecompositionsOfPolynomial( f , onlyone = true )
```
... makes this change locally consistent. But of course it is inconsistent with the surrounding code, which does not put spaces around parenthesis like that...

I wonder whether there is a usable code formatter for Julia? Last time I looked, I couldn't find one.

Reply:
Perhaps add a check that `GAP.Globals.OptionsStack` is empty at this point?

Reply:
Will add that.
I checked for a formatter, and was not able to find one, unfortunately . . .